docs/releasenotes/archive/service/year2018.md

```diff
@@ -5,4 +5,816 @@ sidebar_label: 2018 Archive
+
+---
+## November 6, 2018 (Search)
+
+Update - [Keyboard shortcuts](https://help.sumologic.com/01Start-Here/Keyboard-Shortcuts) are disabled when typing in the [search text box](https://help.sumologic.com/05Search/Get-Started-with-Search/How-to-Use-the-Search-Page).
+
+---
+## November 2, 2018 (Apps)
+
+New - The [Google Cloud Firewall ](https://help.sumologic.com/07Sumo-Logic-Apps/06Google/Google_Cloud_Firewall)app is now available to help you monitor request activity and the effect of your firewall rules. The preconfigured dashboards provide insight into ingress and egress request traffic, including the location of allowed and denied requests, allowed and denied requests over time, and the top networks, subnetworks, and VMs by allowed and denied ingress requests.
+
+New - The Sumo Logic App for [PostgreSQL](https://help.sumologic.com/07Sumo-Logic-Apps/12Databases/PostgreSQL) is now available to help with monitoring your PostgreSQL database. The app provides operational insights into the PostgreSQL database—installed on your local hardware—for real time analysis. The app includes predefined searches and dashboards that allow you to monitor logs and metrics for the database. The logs enable you to monitor database activity, user activity, incoming connections, query execution time, and errors. The metrics allow you to monitor database resource utilization and throughput performance.
+
+---
+## October 31, 2018 (Apps)
+
+Update - An update to the Sumo Logic App for Kubernetes is now available. We updated the searches in the app to work around a bug in the FluentD Kubernetes Plugin. The updated app requires v2.0.0 of FluentD plugin.
+
+Deprecated - The Sumo Logic App for Kubernetes was deprecated September 6, 2019, and replaced with the current [Kubernetes App](https://help.sumologic.com/07Sumo-Logic-Apps/10Containers_and_Orchestration/Kubernetes).
+
+---
+## October 23, 2018 (Apps)
+
+Update - An update to the [Sumo Logic App for Linux](https://help.sumologic.com/07Sumo-Logic-Apps/14Hosts_and_Operating_Systems/Linux) is now available. The update includes fixes to capture failed login events for certain event formats, enhancements to support additional use cases for sudo failed attempts, and updated searches and dashboards.
+
+
+---
+## October 23, 2018 (Security Webinar)
+
+We’re excited to announce that our first Security Webinar recording is available for viewing:
+
+
+---
+## October 23, 2018 (Browser Support)
+
+Update - We now support the [latest two versions](https://help.sumologic.com/01Start-Here/03About-Sumo-Logic/System-Requirements/Supported-Browsers) of the following browsers:
+
+    •    Chrome latest two versions
+    •    Firefox latest two versions
+    •    Safari latest two versions
+    •    Microsoft Edge latest two versions
```

Comment:
Pls fix formatting -- it's rendering as code 
<img width="686" alt="Screen Shot 2022-09-30 at 8 30 32 AM" src="https://user-images.githubusercontent.com/56411016/193305124-eb5def01-7dc2-4c93-8eab-080331105eb4.png">

Reply:
Fixed this.